### ISSUE TITLE
Difference between loss and Train Net loss 

### ISSUE THREAD

**Issue by 12rohanb** [BVLC/caffe#2864]
I'm using caffe to train a CNN with a Euclidean loss layer at the bottom, and my solver.prototxt file configured to display every 100 iterations. I see something like this,

Iteration 4400, loss = 0
I0805 11:10:16.976716 1936085760 solver.cpp:229]     Train net output #0: loss = 2.92436 (* 1 = 2.92436 loss)

I'm confused as to what the difference between the Iteration loss and Train net loss is. Usually the iteration loss is very small (around 0) and the Train net output loss is a bit larger. Can somebody please clarify?

**Comment by seanbell** [comment]
[Contributing guidelines](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md):
> Please do not post usage, installation, or modeling questions, or other requests for help to Issues. Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.

**Comment by 12rohanb** [comment]
I apologize, but the issue on the caffe-users list was actually already brought up by another user, who never received a response. I've bumped the topic but still not received a response, which is why I've posted here.

**Comment by longjon** [comment]
Sorry you have not had much luck on the mailing list, but that does not change the purpose of this tracker.

**Comment by 12rohanb** [comment]
Ok I understand, sorry.

In case anyone can answer the question, I'll leave links to my post on the google group and on stackoverflow

https://groups.google.com/forum/#!topic/caffe-users/WEhQ92s9Vus
http://stackoverflow.com/questions/31840488/caffe-iteration-loss-versus-train-net-loss
